platform/viewer/src/routes/Mode/Mode.tsx

```diff
@@ -38,19 +42,45 @@ async function defaultRouteInit({
 
   unsubscriptions.push(instanceAddedUnsubscribe);
 
-  const { unsubscribe: seriesAddedUnsubscribe } = DicomMetadataStore.subscribe(
```

Comment:
@sedghi  The old workflow here was buggy.  The problem is that the series retrieve ordering is non-deterministic on SOME back ends.  As long as the series come in in the ordering required by the hanging protocol, the study will hang correctly.  However, if the series comes in a different order, then the match can fail, and it won't retry that failure, so you don't get to see anything.